demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import (
+    RelatedFile,
+    RelatedFileType,
+)
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class PackFilesValidator(BaseValidator[ContentTypes]):
+    error_code = "PA128"
+    description = "Checks for required pack files"
+    rationale = "These files are required by the platform."
```

Comment:
not sure the rational is accurate. 
.secrets_ignore and .pack-ignore are required in order to maintain content repo, not the platform.